src/vaadin-confirm-dialog.html

```diff
@@ -212,6 +213,10 @@ <h3 class="header">[[header]]</h3>
               type: String,
               value: 'tertiary'
             },
+            /**
+             * Theme to apply to the overlay element
+             */
+            theme: String,
```

Comment:
`theme` property shouldn't be explicitly added here since it is already inherited from [`ThemableMixin`](https://github.com/vaadin/vaadin-themable-mixin/blob/master/vaadin-themable-mixin.html) (which inherits [`ThemePropertyMixin`](https://github.com/vaadin/vaadin-themable-mixin/blob/master/vaadin-theme-property-mixin.html) since `vaadin-themable-mixin` [v1.4.0](https://github.com/vaadin/vaadin-themable-mixin/releases/tag/v1.4.0)).

The dependency version of `vaadin-themable-mixin` in [bower.json](https://github.com/vaadin/vaadin-confirm-dialog/blob/master/bower.json#L29) should probably be explicitly bumped to current latest `^1.5.1` even though the current `^1.2.0` should still grab the latest `1.x.x`.

Reply:
Removed `theme` and bumped `vaadin-themable-mixin` to `^1.4.0` (as it is the lower version required by this change)